Customizing shell for Lassen.

The following have been reloaded with a version change:
  1) base-gcc/8.3.1 => base-gcc/11.2.1


Lmod is automatically replacing "clang/ibm-16.0.6-cuda-11.2.0-gcc-8.3.1" with
"gcc/10.2.1".


Due to MODULEPATH changes, the following have been reloaded:
  1) spectrum-mpi/rolling-release

Activating 'timing.main' environment for '/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/bin/conda'.
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/scalability_test
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/scalability_test
Driver directory: /p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run
Thu Aug 14 23:23:54 PDT 2025
Running parallel timing tests...
* Running prediction-scalability test in scalability_test.
** Running prediction-scalability_np1 on 1 ranks.
++ runoptions='-n 1'
++ [[ ! -z '' ]]
++ jsrun -g 1 -a 1 -n 1 bash /p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/emirge/mirgecom/scripts/lassen-parallel-spawner.sh python -u -O -m mpi4py driver.py -c prediction-scalability_np1 -g /p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/scalability_test/log_data -i run_params_np1.yaml --log --lazy
Custom casename 'prediction-scalability_np1'
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 8, in <module>
    main()
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 320, in main
    run_command_line(args)
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 53, in run_command_line
    run_path(sys.argv[0], run_name="__main__")
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 53, in <module>
    from mirgecom.simutil import ApplicationOptionsError
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/simutil.py", line 88, in <module>
    import grudge.op as op
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/grudge/grudge/__init__.py", line 23, in <module>
    from grudge.discretization import (
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/grudge/grudge/discretization.py", line 50, in <module>
    from meshmode.discretization.poly_element import (
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/__init__.py", line 39, in <module>
    from meshmode.mesh.tools import AffineMap  # noqa: F401
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/mesh/__init__.py", line 47, in <module>
    from meshmode.mesh.tools import AffineMap, optional_array_equal
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/mesh/tools.py", line 31, in <module>
    from modepy.tools import hypercube_submesh
ImportError: cannot import name 'hypercube_submesh' from 'modepy.tools' (/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/modepy/modepy/tools.py)
++ return_code=1
++ set +x
mv: cannot stat ‘viz_data’: No such file or directory
mv: cannot stat ‘restart_data’: No such file or directory
**  scalability_test/prediction-scalability_np1 failed.
** Running prediction-scalability_np2 on 2 ranks.
++ runoptions='-n 2'
++ [[ ! -z '' ]]
++ jsrun -g 1 -a 1 -n 2 bash /p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/emirge/mirgecom/scripts/lassen-parallel-spawner.sh python -u -O -m mpi4py driver.py -c prediction-scalability_np2 -g /p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/scalability_test/log_data -i run_params_np2.yaml --log --lazy
Custom casename 'prediction-scalability_np2'
Custom casename 'prediction-scalability_np2'
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 8, in <module>
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 8, in <module>
    main()
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 320, in main
    main()
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 320, in main
    run_command_line(args)
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 53, in run_command_line
    run_command_line(args)
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 53, in run_command_line
    run_path(sys.argv[0], run_name="__main__")
    run_path(sys.argv[0], run_name="__main__")
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 53, in <module>
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 53, in <module>
    from mirgecom.simutil import ApplicationOptionsError
    from mirgecom.simutil import ApplicationOptionsError
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/simutil.py", line 88, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/simutil.py", line 88, in <module>
    import grudge.op as op
    import grudge.op as op
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/grudge/grudge/__init__.py", line 23, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/grudge/grudge/__init__.py", line 23, in <module>
    from grudge.discretization import (
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/grudge/grudge/discretization.py", line 50, in <module>
    from grudge.discretization import (
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/grudge/grudge/discretization.py", line 50, in <module>
    from meshmode.discretization.poly_element import (
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/__init__.py", line 39, in <module>
    from meshmode.discretization.poly_element import (
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/__init__.py", line 39, in <module>
    from meshmode.mesh.tools import AffineMap  # noqa: F401
    from meshmode.mesh.tools import AffineMap  # noqa: F401
      ^^^^^^^^^^^^^^^  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/mesh/__init__.py", line 47, in <module>
^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/mesh/__init__.py", line 47, in <module>
    from meshmode.mesh.tools import AffineMap, optional_array_equal
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/mesh/tools.py", line 31, in <module>
    from meshmode.mesh.tools import AffineMap, optional_array_equal
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/mesh/tools.py", line 31, in <module>
    from modepy.tools import hypercube_submesh
    from modepy.tools import hypercube_submesh
ImportError: cannot import name 'hypercube_submesh' from 'modepy.tools' (/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/modepy/modepy/tools.py)ImportError: cannot import name 'hypercube_submesh' from 'modepy.tools' (/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/modepy/modepy/tools.py)

++ return_code=1
++ set +x
mv: cannot stat ‘viz_data’: No such file or directory
mv: cannot stat ‘restart_data’: No such file or directory
**  scalability_test/prediction-scalability_np2 failed.
** Running prediction-scalability_np4 on 4 ranks.
++ runoptions='-n 4'
++ [[ ! -z '' ]]
++ jsrun -g 1 -a 1 -n 4 bash /p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/emirge/mirgecom/scripts/lassen-parallel-spawner.sh python -u -O -m mpi4py driver.py -c prediction-scalability_np4 -g /p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/scalability_test/log_data -i run_params_np4.yaml --log --lazy
Custom casename 'prediction-scalability_np4'
Custom casename 'prediction-scalability_np4'
Custom casename 'prediction-scalability_np4'
Custom casename 'prediction-scalability_np4'
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 8, in <module>
    main()
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 320, in main
    run_command_line(args)
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 53, in run_command_line
    run_path(sys.argv[0], run_name="__main__")
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 53, in <module>
    from mirgecom.simutil import ApplicationOptionsError
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/simutil.py", line 88, in <module>
    import grudge.op as op
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/grudge/grudge/__init__.py", line 23, in <module>
    from grudge.discretization import (
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/grudge/grudge/discretization.py", line 50, in <module>
    from meshmode.discretization.poly_element import (
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/__init__.py", line 39, in <module>
    from meshmode.mesh.tools import AffineMap  # noqa: F401
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/mesh/__init__.py", line 47, in <module>
    from meshmode.mesh.tools import AffineMap, optional_array_equal
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/mesh/tools.py", line 31, in <module>
    from modepy.tools import hypercube_submesh
ImportError: cannot import name 'hypercube_submesh' from 'modepy.tools' (/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/modepy/modepy/tools.py)
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 8, in <module>
    main()
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 320, in main
    run_command_line(args)
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 53, in run_command_line
    run_path(sys.argv[0], run_name="__main__")
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 53, in <module>
    from mirgecom.simutil import ApplicationOptionsError
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/simutil.py", line 88, in <module>
    import grudge.op as op
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/grudge/grudge/__init__.py", line 23, in <module>
    from grudge.discretization import (
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/grudge/grudge/discretization.py", line 50, in <module>
    from meshmode.discretization.poly_element import (
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/__init__.py", line 39, in <module>
    from meshmode.mesh.tools import AffineMap  # noqa: F401
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/mesh/__init__.py", line 47, in <module>
    from meshmode.mesh.tools import AffineMap, optional_array_equal
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/mesh/tools.py", line 31, in <module>
    from modepy.tools import hypercube_submesh
ImportError: cannot import name 'hypercube_submesh' from 'modepy.tools' (/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/modepy/modepy/tools.py)
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 8, in <module>
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/__main__.py", line 8, in <module>
    main()
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 320, in main
    main()
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 320, in main
    run_command_line(args)
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 53, in run_command_line
    run_command_line(args)
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/miniforge3/envs/timing.main/lib/python3.12/site-packages/mpi4py/run.py", line 53, in run_command_line
    run_path(sys.argv[0], run_name="__main__")
  File "<frozen runpy>", line 287, in run_path
    run_path(sys.argv[0], run_name="__main__")
  File "<frozen runpy>", line 287, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 53, in <module>
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "driver.py", line 53, in <module>
    from mirgecom.simutil import ApplicationOptionsError
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/simutil.py", line 88, in <module>
    from mirgecom.simutil import ApplicationOptionsError
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/mirgecom/mirgecom/simutil.py", line 88, in <module>
    import grudge.op as op
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/grudge/grudge/__init__.py", line 23, in <module>
    import grudge.op as op
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/grudge/grudge/__init__.py", line 23, in <module>
    from grudge.discretization import (
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/grudge/grudge/discretization.py", line 50, in <module>
    from meshmode.discretization.poly_element import (
    from grudge.discretization import (
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/grudge/grudge/discretization.py", line 50, in <module>
    from meshmode.discretization.poly_element import (
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/__init__.py", line 39, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/__init__.py", line 39, in <module>
    from meshmode.mesh.tools import AffineMap  # noqa: F401
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    from meshmode.mesh.tools import AffineMap  # noqa: F401
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/mesh/__init__.py", line 47, in <module>
    from meshmode.mesh.tools import AffineMap, optional_array_equal
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/mesh/tools.py", line 31, in <module>
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/mesh/__init__.py", line 47, in <module>
    from meshmode.mesh.tools import AffineMap, optional_array_equal
  File "/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/meshmode/meshmode/mesh/tools.py", line 31, in <module>
    from modepy.tools import hypercube_submesh
ImportError: cannot import name 'hypercube_submesh' from 'modepy.tools' (/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/modepy/modepy/tools.py)
    from modepy.tools import hypercube_submesh
ImportError: cannot import name 'hypercube_submesh' from 'modepy.tools' (/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/emirge/modepy/modepy/tools.py)
++ return_code=1
++ set +x
mv: cannot stat ‘viz_data’: No such file or directory
mv: cannot stat ‘restart_data’: No such file or directory
**  scalability_test/prediction-scalability_np4 failed.
Thu Aug 14 23:24:04 PDT 2025
Scaling/timing tests done.
Passing tests: 
Failing tests:  scalability_test/prediction-scalability_np1 scalability_test/prediction-scalability_np2 scalability_test/prediction-scalability_np4

------------------------------------------------------------
Sender: LSF System <lsfadmin@lassen710>
Subject: Job 7705289: <scale4> in cluster <lassen> Exited

Job <scale4> was submitted from host <lassen709> by user <mtcampbe> in cluster <lassen> at Thu Aug 14 23:22:59 2025
Job was executed on host(s) <1*lassen710>, in queue <pbatch>, as user <mtcampbe> in cluster <lassen> at Thu Aug 14 23:23:46 2025
                            <40*lassen381>
</g/g17/mtcampbe> was used as the home directory.
</p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/scalability_test> was used as the working directory.
Started at Thu Aug 14 23:23:46 2025
Terminated at Thu Aug 14 23:24:05 2025
Results reported at Thu Aug 14 23:24:05 2025

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
#!/bin/bash
#BSUB -nnodes 1
#BSUB -G uiuc
#BSUB -W 180
#BSUB -J scale4
#BSUB -q pbatch
#BSUB -o scal4.txt

source ../emirge/config/activate_env.sh
source ../emirge/mirgecom/scripts/mirge-testing-env.sh
source ../scripts/multi_scalability.sh -p ../ -n 4



------------------------------------------------------------

Exited with exit code 3.

Resource usage summary:

    CPU time :                                   0.18 sec.
    Max Memory :                                 -
    Average Memory :                             -
    Total Requested Memory :                     -
    Delta Memory :                               -
    Max Swap :                                   -
    Max Processes :                              -
    Max Threads :                                -
    Run time :                                   18 sec.
    Turnaround time :                            66 sec.

The output (if any) is above this job summary.

